scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -2348,7 +2606,7 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
   def enumerators(): List[Enumerator] = {
     val enums = new ListBuffer[Enumerator]
     enums ++= enumerator(isFirst = true)
-    while (token.is[StatSep]) {
+    while (token.is[StatSep] && !ahead(token.is[Indentation.Outdent])) {
```

Comment:
Should we add Outdent to StatSep ?

Reply:
no because it is not StatSep, similarly ro Brace.